examples/arm64/main.tf

```diff
@@ -66,4 +66,7 @@ module "this" {
   worker_pool_id    = random_pet.this.id
   ami_id            = data.aws_ami.this.id
   ec2_instance_type = "t4g.micro"
+  spacelift_api_key_id = var.spacelift_api_key_id
```

Comment:
The arguments should be sorted alphabetically, and the file formatted using the terraform fmt command for consistency.

Reply:
Done